### PR TITLE
Cleanup and fixes for Android REST API docs

### PR DESCRIPTION
**Pull request by jamesonwilliams**
Resolves: https://github.com/aws-amplify/docs/issues/2700

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by rjuliano — APPROVED** [review]
🚢  it!!!

File: docs/guides/api-rest/fragments/android/express-api-call.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+    val options = RestOptions.builder()
+        .addPath("/items")
+        .build()
 
     Amplify.API.get(options,
-            restResponse -> Log.i("MyAmplifyApp", restResponse.toString()),
-            apiFailure -> Log.e("MyAmplifyApp", apiFailure.getMessage(), apiFailure)
-    );
+        { Log.i("MyAmplifyApp", "GET succeeded: $it") },
+        { Log.e("MyAmplifyApp", "GET failed." it) }
```
Comment (jamesonwilliams) [inline docs/guides/api-rest/fragments/android/express-api-call.md:11]:
```suggestion
        { Log.e("MyAmplifyApp", "GET failed.", it) }
```

**Comment by jamesonwilliams** [comment]
@aws-amplify/developer-advocates, do the changes under `./guides` gain your approval?

**Comment by dabit3** [comment]
@jamesonwilliams LGTM

**Review by dabit3 — APPROVED** [review]

**Pull request merged**